### PR TITLE
Update changelog for release 1.0.0.

### PR DESCRIPTION
**Pull request by amjith**
@dbcli/pgcli-core This is a big release. We're going 1.0. 

Please review the changelog and add/edit the entries. 

I'd also like people to take the latest master for a spin and raise any issues you might see. 

I'd like to keep this PR open for a few days before I get input from most of the core team before I do the release. 

Thank you!

### CODE REVIEW

**Comment by stuartquin** [comment]
Nice one!
Will be using it for the next few days on Ubuntu 15.10 and 14.04

File: changelog.rst [hunk 1]
```diff
@@ -418,3 +464,17 @@ Improvements:
 .. _`Stuart Quin`: https://github.com/stuartquin
 .. _`Tamas Boros`: https://github.com/TamasNo1
 .. _`Jacek Wielemborek`: https://github.com/d33tah
+.. _`Rodrigo Ramírez Norambuena`: https://github.com/roramirez
+.. _`Anthony Lai`: https://github.com/AnyBucket
```
Comment (ajlai) [inline changelog.rst]: Thanks for doing this @amjith, super excited for the release!
My GH is https://github.com/ajlai though, can we change it in this line? Thanks!

**Comment by auvipy** [comment]
awesome!! when it will be released ?

**Comment by rgrinberg** [comment]
I'm also excited to see a release!

**Comment by amjith** [comment]
@dbcli/pgcli-core Can you please take one final look? I'm going to merge this release a new version asap. Pending approval of #534.

**Comment by j-bennet** [comment]
All looks good.

**Comment by stuartquin** [comment]
Been using it regularly and found no issues, I say go for it :+1:

**Pull request merged**